keps/693-multikueue/README.md

```diff
@@ -223,7 +241,59 @@ If there is enough of global quota, the unknown admitted workloads would be re-a
 the management cluster. If not, some workloads will be preempted to meet the global quota.
 In case of duplicates, all but one of them will be removed.
 
-#### Follow ups ideas
+#### Garbage Collector
+
+Will monitor the Workloads in the remote clusters to detect and remove
+those that ware created by MultiKueue but were not deleted by the Workload Controller 
+in the normal flow, due to a temporary connectivity outage to the remote cluster.
+
+### Jobs abstraction
+
+In order to work with different types of jobs the MultiKueue Controller will use two abstraction
+interfaces:
+
+```go
+// MultiKueueAdapter interface needed for MultiKueue job delegation.
+type MultiKueueAdapter interface {
```

Comment:
Let's not put the whole interface here, just the overall intent and list the important operations (sync, delete)

Reply:
done